packages/logger/src/formatters/ConsoleFormatter.ts

```diff
@@ -0,0 +1,77 @@
+import chalk from 'chalk';
+import {info, warning, error} from 'log-symbols';
+import prettyMs from 'pretty-ms';
+
+import {LogLevel, FormatEntry, Formatter} from '../types';
+
+export class ConsoleFormatter implements Formatter {
+  private startTime: [number, number];
+
+  constructor() {
+    this.startTime = process.hrtime();
+  }
+
+  next(entry: FormatEntry) {
+    const timeDiff = process.hrtime(this.startTime);
+    const msDiff = timeDiff[0] * 1000 + timeDiff[1] / 1000;
```

Comment:
This feels like a totally different level of abstraction from the rest of the method to me. I'd probably at least kill the literals in favour of named constants. It's fine for now but once we have a couple different formatters we might also want to pull it out into a utility function that we can use in all of them. 

```
const timeDiff = msDiff(this.startTime);
```

or even

```
const timestamp = timestampFrom(this.startTime);
```